app/furniture/marketplace/order.rb

```diff
@@ -11,14 +11,22 @@ class Order < Record
     has_many :ordered_products, inverse_of: :order, foreign_key: :cart_id
     has_many :products, through: :ordered_products, inverse_of: :orders
 
+    attribute :delivery_address, :string
+
     enum status: {
       paid: "paid"
     }
 
-    def price_total
+    def product_total
       ordered_products.sum(0) do |ordered_product|
         ordered_product.product.price * ordered_product.quantity
       end
     end
+
+    delegate :delivery_fee, to: :marketplace
+
+    def price_total
+      product_total + delivery_fee
+    end
```

Comment:
Add some tests!

Reply:
done!

Reply:
The tests look like they're covering `Cart#price_total` and `Cart#product_total`, not `Order#price_total` or `Order#product_total`